eng/scripts/Publish-SwiftMirrorRedirectReadme.ps1

```diff
@@ -0,0 +1,56 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $GitSourcePath,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $GitDestinationPath,
+
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string] $UpstreamBranch = "master",
```

Comment:
Should these be updated to be main?

Reply:
It seems like the swift package manager has no assumptions here, so yes. That said, while branch based dependencies aren't recommended, we are technically removing one possible swift dependency mode ([docs](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html)) with this change. @tjprescott any thoughts on this or are we ok enforcing version/commit based dependencies?